client/src/modules/list/index.jsx

```diff
@@ -153,37 +156,40 @@ class List extends Component {
                     {isMember && <InputBar />}
                   </ItemsContainer>
                   <ItemsContainer
-                    archived
                     isMember={isMember}
                     items={doneItems}
+                    ordered
```

Comment:
It was here since the beginning but as an archived prop I changed it to ordered because now the archived word is limited to real archive functionality. It is passed to the second itemsConteiner and it is always ordered in this case.